website/server/extractDocs.js

```diff
@@ -206,30 +208,140 @@ function componentsToMarkdown(type, json, filepath, idx, styles) {
 
 let componentCount;
 
+function getTypedef(filepath, fileContents, json) {
+  let typedefDocgen;
+  try {
+    typedefDocgen = docgen.parse(
+      fileContents,
+      docgenHelpers.findExportedType,
+      [docgenHelpers.typedefHandler]
+    ).map((type) => type.typedef);
+  } catch (e) {
+    // Ignore errors due to missing exported type definitions
+    if (e.message.indexOf(docgen.ERROR_MISSING_DEFINITION) !== -1) {
+      console.error('Cannot parse file', filepath, e);
+    }
```

Comment:
We ignore only this one. There was a thread with the react-docgen core contributors on how to handle these types. However if an API doesn't have a type in it, this throws and that error is okay because it's fine to not have exported types in a class.